modules/core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/security/impl/ACLImpl.java

```diff
@@ -56,6 +57,9 @@
     private final boolean isReadOnly;
 
     public ACLImpl(String name, boolean isReadOnly) {
+        if(name==null) {
```

Comment:
There should be a space before `(` and around `==`.
But instead of these 3 lines please use `Objects.requireNonNull(name, "ACL name can't be null")`. It won't throw a `NuxeoException` but that's not a problem for this kind of check. Unit test will need to be updated of course.